opensrp-tasking/src/test/java/org/smartregister/tasking/util/TaskingMapHelperTest.java

```diff
@@ -0,0 +1,147 @@
+        whenNew(SymbolLayer.class).withAnyArguments().thenReturn(symbolLayer);
+        doReturn(INDEX_CASE_LINE_LAYER).when(lineLayer).getId();
+        whenNew(LineLayer.class).withAnyArguments().thenReturn(lineLayer);
+        revealMapHelper = spy(new TaskingMapHelper());
+        ThreadUtils.init(context);
+    }
+
+    @Test
+    public void testGetIndexCase() {
+        FeatureCollection featureCollection = FeatureCollection.fromFeature(Feature.fromJson(feature));
+        assertNotNull(revealMapHelper.getIndexCase(featureCollection));
+    }
+
+    @Test
+    public void testUpdateIndexCaseLayers() throws JSONException {
+        Feature feature = Feature.fromJson(this.feature);
+        Whitebox.setInternalState(revealMapHelper, "indexCaseSource", geoJsonSource);
+        FeatureCollection featureCollection = FeatureCollection.fromFeature(feature);
+        CameraPosition cameraPosition = new CameraPosition.Builder().target(new LatLng()).build();
+        when(mapboxMap.getCameraPosition()).thenReturn(cameraPosition);
+        revealMapHelper.updateIndexCaseLayers(mapboxMap, featureCollection, context);
+        verify(geoJsonSource).setGeoJson(featureArgumentCaptor.capture());
+        assertEquals("Polygon", featureArgumentCaptor.getValue().geometry().type());
+        JSONObject geometry = new JSONObject(featureArgumentCaptor.getValue().geometry().toJson());
+        assertEquals(1, geometry.getJSONArray("coordinates").length());
+        assertEquals(120, geometry.getJSONArray("coordinates").getJSONArray(0).length());
+    }
+
+    @Test
+    public void testAddIndexCaseLayers() {
+        Feature feature = Feature.fromJson(this.feature);
+        FeatureCollection featureCollection = FeatureCollection.fromFeature(feature);
+        doReturn(style).when(mapboxMap).getStyle();
+        Whitebox.setInternalState(revealMapHelper, "indexCaseSource", geoJsonSource);
+        revealMapHelper.addIndexCaseLayers(mapboxMap, context, featureCollection);
+        verify(style, times(2)).addLayer(layerArgumentCaptor.capture());
+        assertEquals(INDEX_CASE_SYMBOL_LAYER, layerArgumentCaptor.getAllValues().get(0).getId());
+        assertEquals(INDEX_CASE_LINE_LAYER, layerArgumentCaptor.getAllValues().get(1).getId());
+    }
+
+    @Test
+    public void testIndexCaseLayersNotAddedWhenIndexCaseIsNull() {
+        Feature feature = Feature.fromJson(this.feature);
+        feature.removeProperty(IS_INDEX_CASE);
+        FeatureCollection featureCollection = FeatureCollection.fromFeature(feature);
+        doReturn(style).when(mapboxMap).getStyle();
+        Whitebox.setInternalState(revealMapHelper, "indexCaseSource", geoJsonSource);
+        revealMapHelper.addIndexCaseLayers(mapboxMap, context, featureCollection);
+        verify(style, never()).addLayer(layerArgumentCaptor.capture());
+        assertNull(revealMapHelper.getIndexCaseLineLayer());
+
+    }
+
+//    @Test
+//    public void testAddCustomLayers() {
```

Comment:
I am not sure of the intention when commenting these out. Could you do something about it? Create an issue, todo, remove, fix